bitnami/zookeeper/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: zookeeper
-version: 5.9.0
+version: 5.9.1
```

Comment:
Please bump the minor version, it is adding a feature

Reply:
done.